Sources/Testing/Traits/Trait.swift

```diff
@@ -12,75 +12,79 @@
 /// test suite.
 ///
 /// The testing library defines a number of traits that can be added to test
-/// functions and to test suites. Developers can define their own traits by
-/// creating types that conform to ``TestTrait`` and/or ``SuiteTrait``.
+/// functions and to test suites. Define your own traits by
+/// creating types that conform to ``TestTrait`` and/or ``SuiteTrait``:
 ///
-/// When creating a custom trait type, the type should conform to ``TestTrait``
-/// if it can be added to test functions, ``SuiteTrait`` if it can be added to
-/// test suites, and both ``TestTrait`` and ``SuiteTrait`` if it can be added to
-/// both test functions _and_ test suites.
+/// - term ``TestTrait``: Conform to this type in traits that you add to test
```

Comment:
_wat?_

Reply:
I don't understand the question. This syntax is a [term list](https://www.swift.org/documentation/docc/formatting-your-documentation-content#Add-Bulleted-Numbered-and-Term-Lists).

Reply:
I didn't know that was even a thing. Huh.